source/_templates/header.html

```diff
@@ -30,7 +30,7 @@
                 <a href="/{{ language }}/apis/">APIs</a>
             </div>
             <div class="ods__documentation-header-nav-item">
-                <a href="/tutorials/en/">{{ gettext('Tutorials') }}</a>
+                <a href="{{ gettext('https://academy.opendatasoft.com/page/homepage') }}" tabindex="0">Academy</a>
```

Comment:
I don't think we want to actually use the translation system for the url. If feasible, we better have an `if` in the template that displays a different link between 'fr' and all the other languages.
Also I think the tabindex is not needed in an `a`.